File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/model/erd/UserVo.java
package org.javamaster.invocationlab.admin.model.erd;

import lombok.Data;

import java.io.Serializable;
import java.util.Set;

/**
 * @author yudong
 */
@Data
public class UserVo implements Serializable {
    private String orgId;
    private String orgName;
    private Set<String> roleIds;
    private String password;
}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/model/erd/UserVo.kt
package org.javamaster.invocationlab.admin.model.erd

import org.javamaster.invocationlab.admin.annos.AllOpen

import java.io.Serializable

/**
 * @author yudong
 */

@AllOpen
class UserVo : Serializable {
    var orgId: String? = null
    var orgName: String? = null
    var roleIds: Set<String>? = null
    var password: String? = null
}
